src/main/webapp/WEB-INF/view/admin.jsp

```diff
@@ -0,0 +1,27 @@
+<!-- 
+  Admin.jsp
+-->
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Admin</title>
+  <link rel="stylesheet" href="/css/main.css">
+</head>
+<body>
+
+  <nav>
+    <a id="navTitle" href="/">CodeU Chat App</a>
```

Comment:
I'd write a TO-DO comment here to say that you'll be replacing this later on.
We usually format these:
`//TO-DO($USERNAME) Some comment here`